vita3k/gui/src/settings.cpp

```diff
@@ -735,6 +736,9 @@ void draw_settings(GuiState &gui, EmuEnvState &emuenv) {
             ImGui::Columns(1);
             break;
         }
+        case Menu::DATE_FORMAT:
+        case Menu::TIME_FORMAT:
+        case Menu::SELECT_INPUT_LANG: break;
```

Comment:
is strange here

Reply:
What do you mean? If I remove it I get a warning
> Enumeration values 'DATE_FORMAT', 'TIME_FORMAT', and 'SELECT_INPUT_LANG' not handled in switch clang(-Wswitch)